### PR TITLE
Delete a order

### PR DESCRIPTION
**Pull request by dangvm-0369**
## Related Tickets:
- [Task#68110](https://edu-redmine.sun-asterisk.vn/issues/68110)
- [Task#68113](https://edu-redmine.sun-asterisk.vn/issues/68113)

## WHAT this PR does?
- FE +BE Delete a order

## Notes Impacted Areas:
- N/A.

## Attachments
Screenshot 2023-09-20 000328

### CODE REVIEW

**Comment by dangvm-0369** [comment]
ready

File: resources/js/Pages/orders/Index.jsx [hunk 1]
```diff
@@ -1,16 +1,54 @@
 import React, { useState } from 'react';
 import Authenticated from '@/Layouts/Authenticated';
 import { Head } from '@inertiajs/inertia-react';
-import Button from '@/Components/Button';
-import {Table, Tag} from 'antd';
+import {Table, Modal, Button, Tag} from 'antd';
```
Comment (ghost) [inline resources/js/Pages/orders/Index.jsx]:
```suggestion
import { Table, Modal, Button, Tag } from 'antd';
```

File: resources/js/Pages/orders/Index.jsx [hunk 2]
```diff
@@ -1,16 +1,54 @@
...
+                    onError: () => {
+                        openNotification('error',
+                            lang.get('strings.Somethings-went-wrong'),
+                            lang.get('strings.Error-When-Update-To-DB')
+                        );
+                    }
+                });
+            },
+            onCancel() {
+                console.log('Cancel');
```
Comment (ghost) [inline resources/js/Pages/orders/Index.jsx]: delete console log...

**Comment by dangvm-0369** [comment]
ready

**Pull request merged**